counterpartylib/lib/blocks.py

```diff
@@ -874,11 +874,21 @@ def follow(db):
         block_index = config.BLOCK_FIRST
     else:
         block_index = util.CURRENT_BLOCK_INDEX + 1
+        version_major, version_minor = util.get_user_version()
+        version_changed = False
```

Comment:
Do we really need to keep track of whether the version actually changed? Can't we just update `user_version` every time?